packages/fxa-auth-server/fxa-oauth-server/lib/routes/authorized-clients/list.js

```diff
@@ -127,10 +127,10 @@ module.exports = {
         return -1;
       }
       // To help provide a deterministic result order to simplify testing, also sort of scope values.
-      if (a.scope < b.scope) {
```

Comment:
These were accidentally sorting in reverse order, which seems needlessly confusing.